src/script/api/script_road.cpp

```diff
@@ -274,7 +274,7 @@ static int32_t LookupWithBuildOnSlopes(::Slope slope, const Array<RoadPartOrient
 		SLOPE_SW,   SLOPE_WSE, SLOPE_WSE};
 	static const uint8_t base_rotates[] = {0, 0, 1, 0, 2, 0, 1, 0, 3, 3, 2, 3, 2, 2, 1};
 
-	if (slope >= (::Slope)lengthof(base_slopes)) {
+	if (slope >= static_cast<::Slope>lengthof(base_slopes)) {
```

Comment:
I guess this only works because the way `lengthof` has been defined in the macro.
This issue happens in multiple places, just search for `>lengthof(`.